boot/bootchain.go

```diff
@@ -47,7 +47,7 @@ type bootChain struct {
 	KernelRevision string   `json:"kernel-revision"`
 	KernelCmdlines []string `json:"kernel-cmdlines"`
 
-	model          *asserts.Model
+	model          *modeenvModelForSealing
```

Comment:
can this be the interface instead of the concrete type?

Reply:
tweaked to an interface under the same name